app/mailers/concerns/addresses.rb

```diff
@@ -0,0 +1,15 @@
+module Addresses
+  include ActiveSupport::Concern
+
+  def prison_mailbox_email
+    Rails.configuration.prison_data[@visit.prisoner.prison_name]['email']
```

Comment:
Just imagine: one day, we can change this to something sensible like `@visit.prison_email`! But maybe not today.